.eslintrc.js

```diff
@@ -29,7 +29,7 @@ module.exports = {
       version: "detect",
     },
   },
-  ignorePatterns: ["assets/**"],
+  ignorePatterns: ["assets/**", "bin/**"],
```

Comment:
We are linting only the `src` folder from the `lint` script: https://github.com/zendesk/copenhagen_theme/blob/ee8ef959a8e9e4a0442616bea94715ef4bc1b257/package.json#L8

I added `bin` to `ignorePatterns` because I was getting errors like `'process' is not defined` in VSCode, but this can actually be solved just by adding the comment `/* eslint-env node */` on top of the file.

 I can change that, but in any case, we are not lining the `bin` files from the CI, they will be linted only the IDE